### PR TITLE
jit pass of adding a fake-quant for weight-only quantizable op.

### PR DESCRIPTION
**Pull request by chenbohua3**

### CODE REVIEW

**Review by qiuxiafei — APPROVED** [review]
LGTM

File: pytorch_blade/pytorch_blade/quantization/pybind_functions.cpp [hunk 1]
```diff
@@ -78,6 +79,136 @@ torch::jit::Value* insert_prim_constant(
...
+// torch_blade::fake_quant
+void add_fake_quant_for_weight(Module& model) {
+  auto g = model.get_method("forward").graph();
+  Symbol sym = Symbol::fromQualString(
+      torch::blade::quantization::custom_fake_quant_name);
+
+  for (auto&& n : g->nodes()) {
+    std::string node_kind_str = n->kind().toQualString();
+    if (weight_only_list.find(node_kind_str) != weight_only_list.end()) {
+      Value* weight_val = n->inputs()[1];
```
[the remark this answers is not in the document]
Comment (chenbohua3) [reply pytorch_blade/pytorch_blade/quantization/pybind_functions.cpp:108]: For layers like `linear`, `conv`, `deconv`, index 1 meets the requirements. I have added a TODO about this.

File: pytorch_blade/torch_blade/config.py [hunk 2]
```diff
@@ -307,6 +322,14 @@ def enable_int8(self, val):
         assert isinstance(val, bool), "enable_int8 should be bool, got {}".format(type(val))
         self._enable_int8 = val
 
+    @property
+    def quantization_type(self):
```
[the remark this answers is not in the document]
Comment (chenbohua3) [reply pytorch_blade/torch_blade/config.py:326]: done

**Review by qiuxiafei — APPROVED** [review]
LGTM

**Pull request merged**